.travis.yml

```diff
@@ -1,42 +1,39 @@
-﻿dist: trusty
-sudo: false
-env:
-  global:
-    - MAKEFLAGS="-j 2"
+﻿os: linux
+dist: bionic
 
 language: perl
+# See https://www.cpan.org/src/
 perl:
   - 5.30
   - 5.28
   - 5.26
   - 5.24
   - 5.22
   - 5.20
-  - 5.18
-  - 5.16
-  - 5.14
-  - 5.12
-  - 5.10
-  - 5.8
```

Comment:
I wasn't sure what to do here.

I had the following options:
- Follow Perl release policy  https://www.cpan.org/src/: 5.28=<
- Follow popular distributions:
    - Debian https://packages.debian.org/jessie/perl: 5.20=<
    - Ubuntu https://packages.ubuntu.com/xenial/perl: 5.22=<

Since I'm a Debian guy, and we should still target as wide as possible, I choose to keep 5.20=<.
Also even if perl maintains only 5.28 and above, security/bug fixes are still back ported to some older versions.

I don't see any point in maintaining below 5.20 since I have to admit having a EOL distribution for a backup system doesn't make much sense to me. I'd be happy to rush those people to upgrade.

I haven't looked at other systems such as the BSD family or any other distributions. Maybe one of you have an idea ?

I'll add links to the debian/ubuntu package page so one can quickly see which versions are still used by distros.